### PR TITLE
Various Fixes and improvements

### PR DESCRIPTION
**Pull request by NadirRoGue**
- Added rendering control based on quanta streaming (client frame request)
- Added exception catching for corrupted circuits handled by CircuitInfo
- When Rockets plugin crashes, Brayns exits with error code rather than crashing
- Rockets broadcast messages to clients, rather than to specific clients, fixing missing responses for web interface

### CODE REVIEW

File: brayns/engine/Engine.cpp [hunk 1]
```diff
@@ -121,4 +130,24 @@ void Engine::addCameraType(const std::string& name,
     _parametersManager.getRenderingParameters().addCamera(name);
     getCamera().setProperties(name, properties);
 }
+
+bool Engine::mustRender()
+{
+    if(_parametersManager.getApplicationParameters().getUseQuantaRenderControl())
+    {
+        // Seems to work even for animations. If not, add check:
```
Comment (tolokoban) [inline brayns/engine/Engine.cpp]: You're right, it seems strange that the `mustRender()` function return true when a simulation is playing but the camera didn't move.
Maybe the player reset the `frameBuffer->numAccumFrames` to 0. Could you figure it out?

**Comment by NadirRoGue** [comment]
retest this please

File: brayns/engine/Engine.cpp (hunk 1 above)
Comment (NadirRoGue) [reply brayns/engine/Engine.cpp]: commiting simulation data marks the scene/model as modified, which triggers the engine to clear the accumulation frames and thus bypassing the quanta control

**Comment by NadirRoGue** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by NadirRoGue** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Pull request merged**